int8[] robotIDs
Pose2DList startPosition
Pose2DList goalPosition
---
Belief path